js/index.js

```diff
@@ -9,7 +9,7 @@ const siteContent = {
     "img-src": "img/logo.png"
   },
   "cta": {
-    "h1": "DOM Is Awesome",
+    "h1": "DOM<br>Is<br> Awesome",
```

Comment:
take a look at this doc and see if that helps you get the intended effect https://developer.mozilla.org/en-US/docs/Web/CSS/white-space